exampleSite/content/projects/hugo-coder.md

```diff
@@ -0,0 +1,42 @@
++++
+title = "Hugo Coder"
+dates = "2018 - 2020"
```

Comment:
To be removed (dates) if we decide this front matter parameter is to specific for this theme.